implementation/src/test/java/io/smallrye/mutiny/operators/UniOnEventTest.java

```diff
@@ -46,7 +46,7 @@ public void testActionsOnItem2() {
                 .onItem().invoke(Item::set)
                 .onFailure().invoke(failure::set)
                 .on().subscribed(subscription::set)
-                .on().termination(() -> terminate.set(true))
```

Comment:
As for the other PR, we need to be sure that we don't break the deprecated method. I would duplicate the test with one using the deprecated method and the new method. Once we remove the old one, we remove the tests (easy... they won't compile anymore ;-))